ta/sims/ta_sims.c

```diff
@@ -136,3 +136,41 @@ TEE_Result sims_get_counter(void *session_context, uint32_t param_types,
 
 	return TEE_SUCCESS;
 }
+
+TEE_Result sims_entry_panic(void *session_context, uint32_t param_types,
+			    TEE_Param params[4])
+{
+	uint32_t ret_orig;
+	TEE_Result res = TEE_SUCCESS;
+	struct sims_session *ctx = (struct sims_session *)session_context;
+
+	if (param_types != TEE_PARAM_TYPES(TEE_PARAM_TYPE_MEMREF_INPUT,
+					   TEE_PARAM_TYPE_NONE,
+					   TEE_PARAM_TYPE_NONE,
+					   TEE_PARAM_TYPE_NONE))
+		return TEE_ERROR_BAD_PARAMETERS;
+
+	if (params[0].memref.buffer && params[0].memref.size) {
+		res = sims_open_ta_session(session_context,
```

Comment:
`sims_open_ta_session()` is introduce in the next change: this breaks build. Looking at these 2 first commits in the series, I would suggest to either squash them into a single commit or swap them in the series.

Reply:
I kept two separate commits, but in case you are not felling comfortable with their actual implementation, I could squash them into a single commit.

Reply:
Fine as is, thanks.